### ISSUE TITLE
Possible merge issue

### ISSUE THREAD

**Issue by PurpleGuitar** [Bible-Translation-Tools/BTT-Writer-Android#63]
*Submitted by John Wood.  Minor edits from Craig.*

[T]here does appear to be an issue with merging:

1. create English Demo 1 project for 3 John. 
2. Add source (i.e. English ULB). 
3. Add content (Especially a bad chunk, like "test 1".) 
4. Export to project file.
5. Delete project. 
6. Create a second project as before, but choose different source (i.e. French ULB). Must update languages to have access to some GL sources. 
7. Add different content than the first time, but in the same chunks. 
8. Export as project file. You'll need to override the name to keep it from overwriting the first export. 
9. Now import the first export. 
10. There may be a conflict (good, expected). Resolve it. 
11. The project should now have two sources, one from each merged project. It doesn't.
12. Re-import the first project export. Expecting another conflict, but none is forthcoming ( is this because git history has already accounted for that change?)

**Comment by OpThomasPrime1** [comment]
For me, the process is failing at step 8. The app tells me it "Failed Backing up." I continued the steps just to see what would happen. Step 11 rang true, after merging, the project had only one source. Step 12 was odd, because it did see a conflict, and asked me to merge, however, there was still only one source on the project, and no trace of the first project export.

Conclusion: There is definitely an issue (or several) here, but I was unable to reproduce all of it

**Comment by mondele** [comment]
So, this issue of only having one source looks more like an unimplemented feature, than a bug. In desktop, it seems to preserve the source of the pre-existing project when merging two projects. Whether this is desired behavior or not, I don't know. I suspect most of our users don't encounter this as an issue.

It looks like the other part of the issue (expecting a conflict upon re-importing) is also wrong, because `git` has already accounted for the differences in the projects and has merged them.

In short, I think this issue can be closed (with possible addition of a feature request to include all, or as many as possible, source texts upon merge.)
